cooperator_portal/models/res_partner.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 Coop IT Easy SC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def write(self, vals):
+        # deleting iban because it is not a field in partner
+        # it will be written afterwards to the partner's res.partner.bank
+        # it is not deleted before because it needs to be validated
+        # by details_form_validate()
```

Comment:
this comment should be improved. it should explain where th `write()` call is made.

Reply:
@huguesdk Please review changes in [4678d8b](https://github.com/coopiteasy/vertical-cooperative/pull/334/commits/4678d8b830feba93c888a9992c821910996c78bf)